### PR TITLE
Add foundry support

### PR DESCRIPTION
**Pull request by montyly**
Fix 230
Related https://github.com/crytic/slither/issues/1007

TODO:
- [ ] Test the support on large projects

### CODE REVIEW

**Review by elopez — COMMENTED** [review]
Hi! Just two small things I noticed on a quick look.

File: scripts/ci_test_foundry.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+curl -L https://foundry.paradigm.xyz | bash
+source ~/.bash_profile
+foundryup
+
+forge init
+
+crytic-compile .
+if [ $? -ne 0 ]
+then
+    echo "hardhat test failed"
```
Comment (gakonst) [inline scripts/ci_test_foundry.sh:17]:
```suggestion
    echo "foundry test failed"
```

**Review by gakonst — APPROVED** [review]

**Comment by 0xalpharush** [comment]
Would it make sense to ignore `src/tests` by default?

**Comment by montyly** [comment]
@0xalpharush, the tests might be useful (ex: for Echidna). But we could add a `--ignore-test` flag, and let the tools decide to include them or not

**Pull request merged**